pkg/controller/namespace/deletion/namespaced_resources_deleter_test.go

```diff
@@ -306,13 +318,13 @@ func testResources() []*metav1.APIResourceList {
 					Name:       "pods",
 					Namespaced: true,
 					Kind:       "Pod",
-					Verbs:      []string{"get", "list", "delete", "deletecollection", "create", "update"},
+					Verbs:      []string{"get", "list", "delete", "deleteCollection", "create", "update"},
```

Comment:
@sttts this should be lowercase... policy treats it as lowercase at https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go#L210 and discovery writes it as lowercase at https://github.com/kubernetes/kubernetes/tree/master/pkg/genericapiserver/endpoints/installer.go#L72